lib/CL/pocl_util.c

```diff
@@ -675,12 +675,19 @@ pocl_create_command_struct (_cl_command_node **cmd,
   return errcode;
 }
 
+/**
+ * @param dev Destination device
+ * @param ev_export_p Optional output parameter for the export event
+ * @param migration_size Max number of bytes to migrate (caller has to read
+ *                       content size from mem->size_buffer if applicable)
```

Comment:
Set to 0 if not applicable?

Reply:
Currently the caller just sets it to mem->size in that case, and a migration_size of 0 causes the migration to behave as if CL_MIGRATE_MEM_OBJECT_CONTENT_UNDEFINED was given, which seemed like the most sensible thing to do. It is after all entirely possible that a kernel produces no output e.g. due to incomplete input data or due to this being an auxiliary buffer that happens to be unneeded for a given invocation.